packages/health-check/src/utils.js

```diff
@@ -1,8 +1,30 @@
+const FormData = require("form-data");
+const got = require("got");
 const ipCheckService = "whatismyip.akamai.com";
 const ipRegex = new RegExp(
   `^(?:25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]\\d|\\d)(?:\\.(?:25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]\\d|\\d)){3}$`
 );
 
+// sectorSize is the skyd sector size
+const sectorSize = 1 << 22; // 40 MiB
+
+// SECOND is a helper constant for defining the number of milliseconds in a
+// second
+const SECOND = 1000;
+
+// defaultBaseSectorRedundancy is the default baseSectorRedundancy defined by
+// skyd
+const defaultBaseSectorRedundancy = 10;
+
+// defaultFanoutRedundancy is the default fanout redundancy defined by skyd
+const defaultFanoutRedundancy = 3;
+
+// siaDockerContainerIP is the local IP of the sia docker container
+const siaDockerContainerIP = "10.10.10.10";
+
+// defaultSiaPort is the default port the sia runs on.
+const defaultSiaPort = "9980";
```

Comment:
TL;DR: Naming consistency.

Why is one of them "default" and the other one not? When I encountered it in the code it made me think that the `siaContainerIP` could be overridden somehow and that if I want to use the similarly overridden Sia port, I should use not `defaultSiaPort` but `siaPort` or something like that.